DiceKeys/State/DiceKeyMemoryStore.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DiceKeyMemoryStore.swift
+//  DiceKeys
+//
+//  Created by Stuart Schechter on 2021/02/08.
+//
+
+import Foundation
+
+final class DiceKeyMemoryStore: ObservableObjectUpdatingOnAllChangesToUserDefaults {
+    static private(set) var singleton = DiceKeyMemoryStore()
+    
+    init(_ diceKeyLoaded: DiceKey? = nil) {
```

Comment:
This constructor defeats the purpose of singleton. Either remove singleton or remove this construction. We can move this to a variable `var diceKeyLoaded: DiceKey?`

Reply:
Except that for previewing/testing we need to have different values within the same process, so while the app uses a singleton, the previews do not.

Is there a standard way to handle it?  I'll put a comment in for now.